.gitignore

```diff
@@ -31,6 +31,7 @@ cluster/local/certs
 *.crt
 *.csr
 _out
+exported-artifacts
```

Comment:
To make sure that audit and coverage artifcats are uploaded to GCP bucket I add files to this directory, is it correct?

Reply:
According to the prow manual it exports the `ARTIFACTS` variable to let the client know where to put it's output: https://github.com/kubernetes/test-infra/blob/master/prow/jobs.md#job-environment-variables

Reply:
ok so I used this env, should be fine then.